theia-ide/generator-theia-extension
# Add @theia/electron dependency to the example electron app

Fixes #29

**Description**
- Added the `@theia/electron` to the example electron app so that the electron app can be properly built.
- Refactored `app-package.json` template to `app-browser-package.json` and `app-electron-package.json`.

**How to Test**
1. clone the repository, and checkout the PR branch.
2. perform `npm link` on the root of the repository.
3. create a temporary folder in your filesystem, and navigate to it.
4. perform `yo theia-extension` to generate the extension (choose a name when prompted).
5. perform `(yarn && yarn rebuild:electron`).
6. perform `(cd electron-app && yarn start)`.
7. ensure that the electron app starts.


Signed-off-by: Vincent Fugnitto <vincent.fugnitto@ericsson.com>
